app/services/notify/renewal_reminder_email_service.rb

```diff
@@ -14,7 +14,7 @@ def template_id
     def notify_options
       {
         email_address: ContactEmailValidatorService.run(@registration),
-        template_id: "6d20d279-ba79-4fe0-9868-fb09c4ae7733",
+        template_id:,
```

Comment:
Do you know why template IDs in TEMPLATE_ID constant and notify_options were different before? What you've done now looks correct, but I'd like to double check if I missed something

Reply:
It's not clear to me why template_id and TEMPLATE_ID were set up that way. I had updated one value but not noticed the other. I think we should try to defined each id once and once only. I'll create a tech debt item to clean all this up.